pkg/util/flags/flags.go

```diff
@@ -50,20 +50,19 @@ func createRequiredFlagError(name string, flags []string) error {
 	return fmt.Errorf("%s are required for %s", strings.Join(flags, ","), name)
 }
 
-func SetLoggingFlags(flags *pflag.FlagSet) error {
+func SetLoggingFlags(flags *pflag.FlagSet) {
 	//Add commandline flags to the flagset. We will always write to stderr
 	//and not to a file by default
 	flags.AddGoFlagSet(flag.CommandLine)
 	if err := flags.Set("logtostderr", "true"); err != nil {
-		return fmt.Errorf("failed to set flag %q. %v", "logtostderr", err)
```

Comment:
> let me know if I'm doing something wrong but if we do panic inside the method unit test is failing. Is there any other way we can do it?
> after changing this [file](https://github.com/rook/rook/pull/5894/files#diff-ccb61ff6dbab7f545f17b4d1d2129f88R58) it is showing error
> 
> ```
> 
> goroutine 1 [running]:
> github.com/rook/rook/pkg/util/flags.SetLoggingFlags(0xc00045e700)
> 	/home/rook/go/src/github.com/rook/rook/pkg/util/flags/flags.go:58 +0xe4
> github.com/rook/rook/cmd/rook/ceph.init.5()
> 	/home/rook/go/src/github.com/rook/rook/cmd/rook/ceph/operator.go:65 +0x4d5
> ```

Either the logtostderr option has to be added to the flags consistently, or we need to suppress this error. For now it's ok to just log the error.

@leseb This flag should only apply to ceph daemons, right? I don't think this flag is available to other storage providers, so maybe it should be set in a different place. We can fix this in a follow up PR.

Reply:
Done.

Reply:
If this gets expanded to `--log-to-stderr=true` then yes it's a Ceph flag.